modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/preloader/GridDhtPartitionSupplier.java

```diff
@@ -486,7 +486,13 @@ else if (iter.isPartitionMissing(p)) {
             reply(topicId, demanderNode, demandMsg, supplyMsg, ctxId);
 
             if (log.isInfoEnabled())
-                log.info("Finished supplying rebalancing [" + supplyRoutineInfo(topicId, nodeId, demandMsg) + "]");
+                log.info("Finished supplying rebalancing [" +
+                    "partitions=" + supplyMsg.infos().size() +
+                    ", entries=" + supplyMsg.infos().values().stream()
+                        .flatMap(entryCollection -> entryCollection.infos().stream())
+                        .filter(entryInfo -> entryInfo.value() != null)
+                        .count() +
```

Comment:
`testRebalanceLog` will fail if you set `IgniteConfiguration#rebalanceBatchSize` lesser than entries count